### PR TITLE
Rancher 1154 Create deleteTenant job for tenantManagement group

### PR DESCRIPTION
**Pull request by renatsafiulin**

### CODE REVIEW

**Review by aatoyan — APPROVED** [review]

File: src/org/folio/models/InstallRequestParams.groovy [hunk 1]
```diff
@@ -51,6 +54,16 @@ class InstallRequestParams implements Cloneable {
         return this
     }
 
+  /**
+   * Chainable setter for the purge flag.
```
Comment (eldiiar-duishenaliev) [inline src/org/folio/models/InstallRequestParams.groovy:58]: am I missing something, can't see any usage of withPurge property...?

File: pipelines/folioRancher/folioDevTools/tenantManagement/deleteTenant/Jenkinsfile [hunk 2]
```diff
@@ -1 +1,64 @@
...
+
+Main main = new Main(this, namespace.getDomains()['okapi'], namespace.getSuperTenant())
+
+if (!main.isTenantExist(params.TENANT_ID)) {
+  currentBuild.result = 'ABORTED'
+  error('Tenant does not exist, stopping the pipeline')
+}
+
+OkapiTenant tenant = new OkapiTenant(params.TENANT_ID)
+  .withInstallJson(main.getInstallJson(params.TENANT_ID, 'disable'))
```
Comment (eldiiar-duishenaliev) [inline pipelines/folioRancher/folioDevTools/tenantManagement/deleteTenant/Jenkinsfile:37]: Somewhere over here withPurge should be used...

File: pipelines/folioRancher/folioDevTools/tenantManagement/deleteTenant/Jenkinsfile [hunk 3]
```diff
@@ -1 +1,64 @@
-//TBD
+#!groovy
+
+@Library('pipelines-shared-library') _
```
Comment (eldiiar-duishenaliev) [inline pipelines/folioRancher/folioDevTools/tenantManagement/deleteTenant/Jenkinsfile:3]: Usual stuff to 741...

File: pipelines/folioRancher/folioDevTools/tenantManagement/deleteTenant/Jenkinsfile [hunk 4]
```diff
@@ -1 +1,64 @@
...
+      } catch (exception) {
+        println(exception)
+        error(exception.getMessage())
+      } finally {
+        stage('Cleanup') {
+          cleanWs notFailBuild: true
+        }
+      }
+    }
+  }
```
Comment (OHaimanov) [inline pipelines/folioRancher/folioDevTools/tenantManagement/deleteTenant/Jenkinsfile:62]: Please fix here indenting and remove empty lines

**Review by OHaimanov — APPROVED** [review]

**Pull request merged**